chatbot.py

```diff
@@ -1,104 +1,144 @@
 import utilities
 import databaseconnect
-import googleMapsApiModule  
+import googleMapsApiModule
+from enum import Enum, auto
 import logging
 import logger_config
+
 location_dict = {"origin": "null", "destination": "null"}
 
 log = logging.getLogger(__name__)
-log.info('Entered module: %s' % __name__)
+log.info("Entered module: %s" % __name__)
+
+
+class LearnResponse(Enum):
+    MESSAGE = auto()
+    TRAIN_ME = auto()
+    ORIGIN = auto()
+    DESTINATION = auto()
 
 
 @logger_config.logger
 def setup():
     utilities.setup_nltk()
-    logging.debug('NLTK setup completed')
-    clf = utilities.classify_model()
-    logging.debug('Classification model ready')
+    logging.debug("NLTK setup completed")
+    clf = utilities.classify_model_adv(model="rf")
```

Comment:
@eaglewarrior Minor suggestion here. Since the default value for the parameter is 'rf', this does not need the value to be passed explicitly. It's not a real blocker so we can go ahead with this change as it is.